Xamarin/SSW.Rewards/SSW.Rewards/Constants.cs

```diff
@@ -3,7 +3,7 @@
     public class Constants
```

Comment:
is it possible to do some sort of config file for all these constants instead?

Reply:
In theory we can use this: https://mobilebuildtools.com/

We had it in an earlier version, but it doesn't seem to work anymore. Shame, because it supposedly plays nice with pipelines and workflows too.

We should add it to the backlog as tech debt. I don't think it's a priority for now.